src/main/resources/application.conf

```diff
@@ -199,32 +199,28 @@ mantis {
   }
 
   consensus {
-    # FIXME this was moved here from the `mining` section
     active-timeout = 5 seconds
 
     # New timeout to replace `ommer-pool-query-timeout` for uses outside ethash mining
     get-transaction-from-pool-timeout = 5.seconds
 
-    # Moved from the `mining` section
     # Miner's coinbase address
+    # Also used in non-Ethash consensus.
```

Comment:
I can imagine consensus algorithms that do not require it. What do you think about putting it in consensus-specific config? Like `ethash.coinbase` or `atomix-raft.coinbase`

Reply:
🤔 I agree, but then again if a consensus protocol doesn't specify block rewards then this setting is just ignored. So I think it's fine to leave it here.